src/core/opamFilename.ml

```diff
@@ -9,6 +9,16 @@
 (*                                                                        *)
 (**************************************************************************)
 
+let might_escape ~sep path =
+  let sep =
+    match sep with
+    | `Unix -> "/"
+    | `Windows -> "\\"
```

Comment:
are we sure all the `/` in the given path have been replaced by `\` at this point? If both are present then we should check for both on Windows given they are both valid path separators

Reply:
Updates windows with `/|\' search